bootstrap/_deb_common.sh

```diff
@@ -10,26 +10,37 @@
 #     - 7.8 "wheezy" (x64)
 #     - 8.0 "jessie" (x64)
 
-# virtualenv binary can be found in different packages depending on
-# distro version (#346)
-distro=$(lsb_release -si)
-# 6.0.10 => 60, 14.04 => 1404
-version=$(lsb_release -sr | awk -F '.' '{print $1 $2}')
-if [ "$distro" = "Ubuntu" -a "$version" -ge 1410 ]
-then
-  virtualenv="virtualenv"
-elif [ "$distro" = "Debian" -a "$version" -ge 80 ]
-then
-  virtualenv="virtualenv"
-else
-  virtualenv="python-virtualenv"
-fi
+apt-get update
 
 # dpkg-dev: dpkg-architecture binary necessary to compile M2Crypto, c.f.
 #           #276, https://github.com/martinpaljak/M2Crypto/issues/62,
 #           M2Crypto setup.py:add_multiarch_paths
 
-apt-get update
-apt-get install -y --no-install-recommends \
-  python python-setuptools "$virtualenv" python-dev gcc swig \
-  dialog libaugeas0 libssl-dev libffi-dev ca-certificates dpkg-dev
+common_packages="python python-setuptools python-dev gcc swig dialog libaugeas0 libssl-dev openssl libffi-dev"
+
+if [ "$1" = "no_venv" ]
+then
+  packages="$common_packages"
+else
+
+  # virtualenv binary can be found in different packages depending on
+  # distro version (#346)
+  distro=$(lsb_release -si)
+  # 6.0.10 => 60, 14.04 => 1404
+  version=$(lsb_release -sr | awk -F '.' '{print $1 $2}')
+  if [ "$distro" = "Ubuntu" -a "$version" -ge 1410 ]
+  then
+    virtualenv="virtualenv"
+  elif [ "$distro" = "Debian" -a "$version" -ge 80 ]
+  then
+    virtualenv="virtualenv"
+  else
+    virtualenv="python-virtualenv"
+  fi
+
+  packages="$virtualenv $common_packages"
+fi
+
+apt-get install -y --no-install-recommends $packages
+
+ln -s /usr/include/x86_64-linux-gnu/openssl/opensslconf.h /usr/include/openssl/
```

Comment:
I've tested the client on multiple platforms, but none of them required me to do that symlink.  In fact, you can see that Travis fails... What's the rationale behind that? It should be included as inline comment.